lib/crypto.js

```diff
@@ -647,7 +647,14 @@ exports.setEngine = function setEngine(id, flags) {
   return binding.setEngine(id, flags);
 };
 
-exports.randomBytes = exports.pseudoRandomBytes = randomBytes;
+exports.randomBytes = exports.pseudoRandomBytes = function(size, callback) {
+  const warning = 'randomBytes used as async is deprecated. ' +
+                  'Use randomBytesSync instead.';
```

Comment:
wait... i'm confused now. i don't see a `randomBytesSync()` function - I think I added that before but it's no longer relevant. removing this line.